rules-reviewed/jboss-eap/websphere/WebSphereMQ.windup.xml

```diff
@@ -0,0 +1,1363 @@
+                    <message>Replace with Java EE 6 JMS: `javax.jms.TopicSession`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mq-74ugps">
+            <when>
+                <javaclass references="com.ibm.mq.jms.MQXASession"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQ API." description="A more general, and recommended, approach is to use the XASession class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.XASession`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mq-76hgq">
+            <when>
+                <javaclass references="com.ibm.mq.jms.MQXAQueueSession"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQ API." description="A more general, and recommended, approach is to use the XAQueueSession class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.XAQueueSession`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mq-78hhew">
+            <when>
+                <javaclass references="com.ibm.mq.jms.MQXATopicSession"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQ API." description="A more general, and recommended, approach is to use the XATopicSession class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.XATopicSession`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+
+
+        <!-- SIB JMS -->
+        <rule id="websphere-mq-7Apvvd">
```

Comment:
If they weren't assigned to you, then they really shouldn't be included in your PR because that would confuse the next person :) If you feel like doing them and including them, please make sure they are finished. Thanks!